### PR TITLE
recommend the typical build suggestions for macs

### PR DESCRIPTION
**Pull request by jmtroth0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,12 +83,13 @@ Lines your file is missing will begin with a `-`.
 
 If you want to run FormPlayer in Docker as a service of CommCare HQ, follow these steps from your commcare-hq repository:
 
-1. Start Docker for CommCare HQ as usual, either with `scripts/docker up` (services only) or `scripts/docker runserver` (HQ and services).
-2. Run `scripts/get_webhost` and append its output to `/etc/hosts`. (Or if you run Linux, `scripts/get_webhost | sudo tee -a /etc/hosts`.)
-3. If you run CommCare HQ in Django locally:
+1. Check if you are on a Mac. If you are, don't bother with this. Just follow the [directions above](https://github.com/dimagi/formplayer#building-and-running).
```
Comment (proteusvacuum) [inline README.md]: lol.

File: README.md [hunk 2]
```diff
@@ -25,10 +25,14 @@ To make properties file:
 
     $ cp config/application.example.properties config/application.properties  # Update properties as necessary (the defaults are fine for running locally)
 
+In particular, make sure the `server.port` value is the same as the port from `FORMPLAYER_URL` in commcarehq's `localsettings.py`.
+
 Make sure you have the formplayer database created. You will be asked to provide a password after running this command; assuming you are running formplayer locally, you should use the password for the postgres user associated with the locally-running instance of commcare hq (which can be found in the `DATABASES` section of your `localsettings.py` file).
 
     $ createdb formplayer -U commcarehq -h localhost  # Update connection info as necessary (the defaults are fine for running locally)
 
+If you are running postgres in Docker, you may need to run this in the docker shell, using `./scripts/docker bash postgres` from the commcarehq repository.
```
Comment (proteusvacuum) [inline README.md]: really? I think you should be able to access postgres without having to run this command directly in the container

Comment (jmtroth0) [reply README.md]: I never installed postgres on my machine, so `createdb` only existed in the container

Comment (proteusvacuum) [reply README.md]: ah, makes sense

**Pull request merged**